src/views/Home.vue

```diff
@@ -2,7 +2,16 @@
   <div>
     <div class="text-center mb-4 mx-auto">
       <Container class="d-flex flex-items-center">
-        <div class="flex-auto text-left">
+        <div class="flex-auto text-left d-flex">
+          <router-link :to="{ name: 'timeline' }">
+            <UiButton class="mr-2" style="width: 135px">
```

Comment:
Can you try like this?
```suggestion
            <UiButton class="mr-2 no-wrap">
```

Reply:
Yeah that's better